### PR TITLE
Zq/fix rms norm

### PR DESCRIPTION
**Pull request by NeosZhang**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation and Context

fix gen_output and update test_cases for rms_norm

## Description

fix gen_output and update test_cases for rms_norm

## Checklist
**Before PR**:

- [x] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: diopi_test/python/conformance/diopi_functions.py [hunk 1]
```diff
@@ -5151,8 +5151,8 @@ def rms_norm(input, normalized_shape, weight, bias, eps):
     func = check_function(call)
     size = list(input.size().data)
     out = Tensor(size, input.get_dtype())
-    inv_rms_size = size.copy()
```
Comment (NeosZhang) [inline diopi_test/python/conformance/diopi_functions.py:5154]: inv_rms的size的计算逻辑有误

File: diopi_test/python/conformance/gen_output.py [hunk 2]
```diff
@@ -250,11 +250,19 @@ def rotary_emb(input, cos, sin, conj, interleaved):
         return out
 
     def rms_norm(input, normalized_shape, weight, bias, eps):
-        var = input.to(torch.float32).pow(2).mean(-1, keepdim=True)
-        inv_rms = torch.rsqrt(var + eps)
-        inp = input * inv_rms
-        out = weight * inp
-
+        square = input.pow(2)
```
Comment (NeosZhang) [inline diopi_test/python/conformance/gen_output.py:253]: 添加对normalized和bias的支持

File: impl/ascend/functions_ext/rms_norm.cpp [hunk 3]
```diff
@@ -14,6 +14,8 @@ diopiError_t diopiRMSNorm(diopiContextHandle_t ctx, diopiTensorHandle_t out, dio
     AscendTensor inputTensor(input);
     ASCEND_CHECK_ABORT(1 == normalizedShape.len && normalizedShape.data[0] == inputTensor.shape()[inputTensor.dim() - 1], "normalized shape error!");
     AclOpRunner<2, 2>("RmsNorm", ctx).addInput(input).addInput(weight).setAttr("epsilon", static_cast<float>(eps)).addOutput(out).addOutput(invRms).run();
+    diopiScalar_t oneScalar = constructDiopiScalarT(diopi_dtype_float32, 1.0);
```
Comment (NeosZhang) [inline impl/ascend/functions_ext/rms_norm.cpp:17]: 添加对bias的支持

File: impl/torch/functions/functions_ext.cpp [hunk 4]
```diff
@@ -63,7 +63,7 @@ diopiError_t diopiRMSNorm(diopiContextHandle_t ctx, diopiTensorHandle_t out, dio
     auto atWeight = impl::aten::buildATen(weight);
     auto atBias = impl::aten::buildATen(bias);  // bias在这里实际上没有使用
     ext::ops::rms_norm_forward(atInput, atNormalizedShape, atWeight, eps, atOut, atInvRMS);
-
+    atOut.add_(atBias);
```
Comment (NeosZhang) [inline impl/torch/functions/functions_ext.cpp:66]: 添加对bias的支持

File: impl/ascend/convert_config.yaml [hunk 5]
```diff
@@ -303,3 +303,9 @@
 
 - diopiGatherBackward:
     dtype: (float64)->float32
+
+- diopiRMSNorm:
```
Comment (NeosZhang) [inline impl/ascend/convert_config.yaml:307]: ascend底层算子RmsNorm不支持float64

File: diopi_test/python/conformance/diopi_functions.py [hunk 6]
```diff
@@ -5151,8 +5151,8 @@ def rms_norm(input, normalized_shape, weight, bias, eps):
...
-    inv_rms_size[-1] = 1
+    normalized_dim = list(range(-(len(normalized_shape)), 0))
```
Comment (yangbofun) [inline diopi_test/python/conformance/diopi_functions.py:5154]: 加个assert判断normalized shape与input shape的约束。

File: impl/ascend/device_configs.py [hunk 7]
```diff
@@ -786,6 +786,14 @@
         rtol=1e-3,
         atol_half=1e-2,
         rtol_half=1e-2,
+        tensor_para=dict(
+            args=[
+                {
+                    "ins": ['input'],
+                    "dtype": [Skip(np.float16)],
+                },
```
Comment (yangbofun) [inline impl/ascend/device_configs.py:794]: 查一下fp16不过的原因.

**Review by LeungChiNan — APPROVED** [review]

**Review by hellozmz — APPROVED** [review]

File: diopi_test/python/conformance/diopi_functions.py (hunk 6 above)
Comment (NeosZhang) [reply diopi_test/python/conformance/diopi_functions.py:5155]: done

File: impl/ascend/device_configs.py (hunk 7 above)
Comment (NeosZhang) [reply impl/ascend/device_configs.py:794]: 底层算子在计算某些特殊形状的float16时报错

**Pull request closed without merging**